packages/ember-testing/lib/helpers/pause_test.js

```diff
@@ -57,9 +57,6 @@ export function resumeTest() {
  @public
 */
 export function pauseTest() {
-  // eslint-disable-next-line no-console
-  console.info('Testing paused. Use `resumeTest()` to continue.');
```

Comment:
We don't want to remove this, it is intentionally logging to the console to let developers know how they can resume. Please revert this removal...

Instead of removing this, we should override `console.info` in the specific tests (and confirm it is called properly).

Reply:
Thanks Robert! I figured that outright deleting this part could have been a bit too much.
Overwriting `console.info` should be possible, but what I fear is that the test code which relies on functions such as `pauseTest` will rely more and more on such workarounds. Every occurrence of console.info, console.warn and the like would need to be patched (now and in future tests).

Reply:
It might be a good idea to indirect these through a `@ember/debug` function (which makes it easier to swap out), what do you think @rwjblue?